docs/content/en/docs/concepts/tracing-policy/hooks.md

```diff
@@ -212,6 +212,64 @@ spec:
 This example shows how to use uprobes to hook into the readline function
 running in all the bash shells.
 
+## LSM BPF
+
+LSM BPF programs allow runtime instrumentation of the LSM hooks by privileged
+users to implement system-wide MAC (Mandatory Access Control) and Audit policies
+using eBPF.
+
+List of LSM hooks which can be instrumented can be found in `security/security.c`.
+
+To verify if BPF LSM is available use the following command:
+
+```
+$ cat /boot/config-$(uname -r) | grep BPF_LSM
+CONFIG_BPF_LSM=y
+```
+
+If the output contains `CONFIG_BPF_LSM=y`, BPF LSM is supported.
+Then, if provided above conditions are met, use this command to check if BPF LSM is enabled:
+
+```
+$ cat /sys/kernel/security/lsm
+bpf,lockdown,integrity,apparmor
+```
```

Comment:
should this be part of the lsm detection? I might be wrong here but IIUC the whole thing does not work of 'bpf' is not present in here, right?

Reply:
Yes, exactly you are right! `CONFIG_LSM=y`. Tells us that kernel is compiled with LSM bpf support.  `bpf` in  `/sys/kernel/security/lsm` tells us that LSM is enabled right here, right now.

`bpf.HasLSMPrograms()` tries to load LSM  BPF program. It seems to me, that to load `CONFIG_LSM=y`  is enough. That's  why only TestLSMObjectLoad working in gotests.

Thanks so much! I'll add this check to `bpf.HasLSMPrograms()`!